HelperFunctions.ps1

```diff
@@ -1058,6 +1058,12 @@ function GetAppInfo {
         $alToolExe = Join-Path $alcPath 'altool'
         & /usr/bin/env sudo pwsh -command "& chmod +x $alToolExe"
     }
+    elseif ($isMacOS) {
+        $alcPath = Join-Path $binPath 'darwin'
+        $alToolExe = Join-Path $alcPath 'altool'
+        Write-Host "Setting execute permissions on altool"
+        & chmod +x $alToolExe
```

Comment:
No sudo needed?

Reply:
The current user has permissions to the user's folder, no admin permissions required.